app/views/merchant_items/index.html.erb

```diff
@@ -1,8 +1,39 @@
 <h1>Merchant Items Index Page</h1>
-<% @merchant_items.each do |item| %>
-<div id=<%="item-#{item.id}"%>>
-  <%= link_to item.name, merchant_item_path(@merchant.id, item.id)%>
-  <%= "Status: #{item.status}" %>
-  <%= button_to "Enable", merchant_item_path(@merchant.id, item.id), params: {status: :enabled},method: :patch %>
-  </div>
-<% end %>
+<h2><%= @merchant.name %></h2>
+<hr>
+<center><h3>My Items</h3></center>
+<%= link_to "Create New Item", new_merchant_item_path(@merchant.id) %></p>
+<p>
```

Comment:
does this `<p>` tag belong to the line above it?

Reply:
Hmm I think Anthony added this for a good reason to have the view look right.  I don't know how it works